exercises/practice/transpose/tests/transpose.cairo

```diff
@@ -0,0 +1,174 @@
+use transpose::transpose as transpose_matrix;
+
+#[test]
+fn empty_string() {
+    let mut input: Array<ByteArray> = array![];
+    let expected = array![];
+
+    let output = transpose_matrix(input);
+    assert_eq!(output, expected);
+}
+
+#[test]
+#[ignore]
+fn two_characters_in_a_row() {
+    let mut input: Array<ByteArray> = array!["A1"];
+    let expected = array!["A", "1"];
+
+    let output = transpose_matrix(input);
+    assert_eq!(output, expected);
+}
+
+#[test]
+#[ignore]
+fn two_characters_in_a_column() {
+    let mut input: Array<ByteArray> = array!["A", "1"];
+    let expected = array!["A1"];
+
+    let output = transpose_matrix(input);
+    assert_eq!(output, expected);
+}
+
+#[test]
+#[ignore]
+fn simple() {
+    let mut input: Array<ByteArray> = array!["ABC", "123"];
+    let expected = array!["A1", "B2", "C3"];
```

Comment:
nit: let's format these row values into _rows_, and column values into _columns_. 

You can use `#[cairofmt::skip]` to tell the formatter to not touch this then.
This is similar to what we've done for [minesweeper](https://github.com/exercism/cairo/blob/01caba201497103feb5bdc6b83995420b3672a42/exercises/practice/minesweeper/tests/minesweeper.cairo#L57).

```rust
#[cairofmt::skip]
let input: Array<ByteArray> = array![
    "ABC", 
    "123",
];
#[cairofmt::skip]
let expected = array![
    "A1", 
    "B2",
    "C3",
];
```

Note: apply this change to all tests where the formatter didn't already format the values like this